Kconfig.zephyr

```diff
@@ -360,9 +360,13 @@ config NO_OPTIMIZATIONS
 	help
 	  Compiler optimizations will be set to -O0 independently of other
 	  options.
-
 endchoice
 
+config ENABLE_WARNING_AS_ERRORS
```

Comment:
`ENABLE` is redundant. It should be called something like `COMPILER_WARNINGS_AS_ERRORS`.

Reply:
Fixed